csharp.test/ParquetSharp.Test.csproj

```diff
@@ -1,8 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)'=='Windows_NT'">$(TargetFrameworks);net472</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(GITHUB_ACTIONS)' == 'true'">$(TargetFrameworks);netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
I think that we should avoid unnecessary discrepancy between local testing and CI. Also, there is no benefit from removing other target frameworks for local development and testing, or is it?

Reply:
This is just to avoid getting errors when running the tests locally on a machine that does not have the required runtimes installed. It also avoids building those target frameworks by default, speeding up the build process. And then our CI ensures that those frameworks still build and run without warnings or errors.

FYI I was inspired by what we started doing with [ILGPU](https://github.com/m4rs-mt/ILGPU/blob/v1.2.0/Src/Directory.Build.props).

Reply:
Happy to revert this if we don't feel comfortable with it!

Reply:
> Happy to revert this if we don't feel comfortable with it!

Nah, I think it is ok. I would just add some comment explaining our motivation. Also, I think that `Condition="'$(CI)' == 'true'">` will be better (as it is more universal).

Reply:
Great point!